### PR TITLE
add monitis as t-shirt sponsor

### PR DESCRIPTION
**Pull request by pmoosh**

### CODE REVIEW

File: data/events/2016-siliconvalley.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ sponsors: # List all of your sponsors here along with what level of sponsorship
     level: silver
   - id: extrahop
     level: silver
+  - id: 2015-pagerduty
```
Comment (bridgetkromhout) [inline data/events/2016-siliconvalley.yml]: Can you put this in as pagerduty, not 2015-pagerduty? The 2015 one has the blue border hard-coded.

File: data/events/2016-siliconvalley.yml [hunk 2]
```diff
@@ -37,6 +37,10 @@ sponsors: # List all of your sponsors here along with what level of sponsorship
...
+    level: silver
+  - id: 2016-monitis
```
Comment (bridgetkromhout) [inline data/events/2016-siliconvalley.yml]: Please add the new sponsor as just monitis. We're trying to move away from the dates in favor of a simple one and then updating it as needed with `-before-datestamp`.

**Comment by bridgetkromhout** [comment]
I would prefer monitis not be labeled as 2016, but I'm merging this in the interests of getting it done. Probably will clean up later if/when we do the rest.

**Comment by pmoosh** [comment]
I do whatever needs to be done. I didn't use filenames with prefixes, until I saw them appearing. So if they are not necessary, I will stop using them. Thank you for the merge

**Pull request merged**